rustls/src/client/builder.rs

```diff
@@ -12,7 +12,7 @@ use crate::error::Error;
 use crate::key_log::NoKeyLog;
```

Comment:
Another comment without a good home: I think this commit could also update [`rustls/src/manual/features.rs`](https://github.com/rustls/rustls/blob/main/rustls/src/manual/features.rs) to mention that we support RFC 7250 RPKs thanks to this work.